opencti-platform/opencti-front/src/utils/dashboard.ts

```diff
@@ -19,3 +21,10 @@ export interface DashboardWidgetParameters {
   legend?: boolean
   distributed?: boolean
 }
+
+export const copyPublicLinkUrl = (t: (text: string) => string, uriKey: string) => {
```

Comment:
Is the front part (workspace/public dashboard & co) related to the issue ?

Reply:
nope